server/fleet/software_installer.go

```diff
@@ -444,7 +445,7 @@ type SoftwarePackageOrApp struct {
 	AppStoreID string `json:"app_store_id,omitempty"`
 	// Name is only present for software installer packages.
 	Name string `json:"name,omitempty"`
-	// AutomaticInstallPolicies is only present for Fleet maintained apps
+	// AutomaticInstallPolicies is present for Fleet maintained apps and custom packages
```

Comment:
Are we good on docs updates here?

Reply:
Any issue with the update? :)

Reply:
Will check when back from AFK, sorry.

Reply:
Feedback left for the article change. Was mainly concerned about API docs changes, and those are good to go.